tilelang/utils/pass_diff_hook.py

```
@@ -0,0 +1,198 @@
+    generate_html = _diff_utils["generate_html"]
+    dim = _diff_utils["dim"]
+    reset = _diff_utils["reset"]
+
+    from tilelang.env import env
+
+    mode = env.get_pass_diff_mode() or "terminal"
+
+    # Capture before
+    try:
+        before_script = mod.script()
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)